extensions/shared/styles/gutenberg-base-styles.scss

```diff
@@ -4,3 +4,21 @@
 @import '~@wordpress/base-styles/breakpoints';
 @import '~@wordpress/base-styles/mixins';
 @import '~@wordpress/base-styles/animations';
+
+/**
+ * Following is a fix for column contents overflowing the parent container
+ * when vertically aligned. This has been taken from newer versions of
+ * Gutenberg and WP core to provide the fix for WP 5.3.
+ *
+ * Issue: https://github.com/Automattic/jetpack/issues/14524
+ *
+ * See: https://github.com/WordPress/gutenberg/commit/bef41793a12b76a5220d177a768f8fe56deb5eba
+ * See: https://github.com/WordPress/WordPress/commit/f616ffb9c1a0abcdbe3306fd9e695ef3173a18c6
+ */
+.wp-block-column {
+	&.is-vertically-aligned-top,
+	&.is-vertically-aligned-center,
+	&.is-vertically-aligned-bottom {
+		width: 100%;
+	}
+}
```

Comment:
Although the original issue relates to the Slideshow component, it is a core column block issue, and that is where it has been fixed in core, this seems like the obvious place to put the temporary fix. Let us know if you think there is a better home for it.

This should have no effect on sites that already have the core fix as one will just override the other.